code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -72,7 +72,7 @@
 /obj/machinery/recycler/lumbermill
 	name = "lumbermill saw"
 	desc = "Faster then the cartoons!"
-	emagged = 2 //Always gibs people
+	obj_flags = CAN_BE_HIT | EMAGGED
```

Comment:
Changing behavior here.

Reply:
Yeah, however I don't really care about behavior that, according to the comment on the code, only exists for admemes and is taken advantage of for an away mission that is never going to see the light of day on /tg/